src/types/types.ts

```diff
@@ -702,6 +702,8 @@ export interface EdgeSpendInfo {
   /** @deprecated Use EdgeCurrencyWallet.accelerate instead */
   rbfTxid?: string
   skipChecks?: boolean
+  /** Allow creating transactions with chained nonces even if pending transactions exist */
+  allowChainedPending?: boolean
```

Comment:
This is not what we discussed. Since this is a 100% EVM problem, so it shouldn't be in the core types. It should be in the `otherParams`. So this PR needs to be dropped.